### PR TITLE
feat: react flow

### PR DESCRIPTION
**Pull request by D-B-Hawk**

### CODE REVIEW

File: components/flow/index.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, type FunctionComponent } from 'react';
+import ReactFlow, {
+  addEdge,
+  Edge,
+  Connection,
+  NodeTypes,
+  useNodesState,
+  useEdgesState,
+} from 'reactflow';
+import { GraphNode, type CustomGraphNode } from 'components/graphNode';
```
Comment (CristhianF7) [inline components/flow/index.tsx:10]:
```suggestion
import { GraphNode, type CustomGraphNode } from '../graphNode';
```

File: components/graphNode/graphNode.styled.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  line-height: 16px;
+  letter-spacing: 0.4px;
+`;
+
+export const NodeHandle = styled(Handle)<{ bgColor: string }>`
+  right: -8px;
+  height: 16px;
+  width: 16px;
+  border: none;
+  background-color: ${({ bgColor }) => bgColor};
```
Comment (CristhianF7) [inline components/graphNode/graphNode.styled.ts:22]: Most of the styles are sorted A-Z and I'd like to keep this across the project. Just for readability purposes

**Review by CristhianF7 — DISMISSED** [review]
Thanks @D-B-Hawk, great work. I just left some suggestions, feel free to fix them in the following PR or whatever you think is best. Thank you

File: components/flow/index.tsx (hunk 1 above)
Comment (D-B-Hawk) [reply components/flow/index.tsx:10]: auto import strikes again🙃

**Review by CristhianF7 — APPROVED** [review]

**Pull request merged**